src/custom/state/orders/utils.ts

```diff
@@ -154,27 +155,27 @@ export function isOrderUnfillable(
 
 export function getOrderExecutionPrice(order: Order, price: string, feeAmount: string): Price<Currency, Currency> {
   if (order.kind === OrderKind.SELL) {
-    return new Price(order.inputToken, order.outputToken, order.sellAmount.toString(), price.toString())
+    return new Price(order.inputToken, order.outputToken, getRemainingAmount('sell', order), price.toString())
   }
 
   return new Price(
```

Comment:
not from this PR, but this line could have a comment `// Returns buy order's Price`